src/project-tests/scatter-plot-tests.js

```diff
@@ -84,9 +112,38 @@ export default function createScatterPlotTests() {
                     return new Date(a.getAttribute("data-yvalue")) - new Date(b.getAttribute("data-yvalue"));
                 });
 
-                //check to see if the y locations of the new sorted array are in ascending order
-                for (var i = 0; i < sortedDots.length - 1; ++i) {
-                    FCC_Global.assert.isAtMost(+sortedDots[i].cy.baseVal.value, +sortedDots[i + 1].cy.baseVal.value, "y values don't line up with y locations ");
+                var yAxisQuery = document.querySelector('#y-axis path');
```

Comment:
A lot of the code here looks very similar to what you did for the x-axis. Could the code be shared in a function? Or better, several small functions?